addons/magazinerepack/stringtable.xml

```diff
@@ -132,7 +132,7 @@
         <Key ID="STR_ACE_MagazineRepack_RepackedMagazinesCount">
             <English>%1 Full and %2 Partial</English>
             <French>%1 plein(s) et %2 partiel(s)</French>
-            <German>%1 volle Magazine und 1 Magazin mit %2 Partronen</German>
+            <German>%1 volle Magazine und %2 angefangenes Magazine</German>
```

Comment:
Maybe `Magazin(e)` so it works with "1"? Same for `volle(s)` und `angefangene(s)`.